silhouette/app/com/mohiva/play/silhouette/api/Authorization.scala

```diff
@@ -35,10 +35,10 @@ trait Authorization[I <: Identity] {
    *
    * @param identity The identity to check for.
    * @param request The current request header.
-   * @param lang The current lang.
+   * @param messages The messages for the current language.
    * @return True if the user is authorized, false otherwise.
    */
-  def isAuthorized(identity: I)(implicit request: RequestHeader, lang: Lang): Boolean
+  def isAuthorized(identity: I)(implicit request: RequestHeader, messages: Messages): Boolean
```

Comment:
Is it necessary `Messages` as param?

because it's possible to do that
```scala
class I18nAuthorization (implicit messagesApi: MessagesApi) extends Authorization[User] with I18nSupport {

  def isAuthorized(identity: User)(implicit request: RequestHeader): Unit = {
    val messages: Messages = request2Messages
    ...
  }
}
```